### ISSUE TITLE
Add support for nested fields arguments

### ISSUE THREAD

**Issue by qlient-org** [qlient-org/python-qlient#20]
As eagle258 pointed out on reddit, it is possible to have nested field variables:

```gql
query {
  viewer {
    name
     repositories(last: 5) {
       nodes {
         name
       }
     }
   }
}
```

Although the qlient supports nested field selection, a nested variables selection is yet not possible.

**Comment by dfseifert** [comment]
Implemented with #27

**Comment by ghost** [comment]
@dfseifert 
I can't figure out how to set nested field arguments with `qlient`.
Looking at the source code, it seems that this feature was removed with the `v1.0.0` release.
Is this true, or did I miss something?
